### PR TITLE
update 0.6.0 snapshots

### PR DESCRIPTION
**Pull request by helenlly**
fixed issue#1432

### CODE REVIEW

**Comment by helenlly** [comment]
fixed https://github.com/intel-analytics/analytics-zoo/issues/1432

File: docs/docs/release-download.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## **Release 0.6.0 nightly build**
+
+| | BigDL 0.8.0 | 
+| ------------- | --------- |
+| Spark 2.1.1   | [download](https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/zoo/analytics-zoo-bigdl_0.8.0-spark_2.1.1/)|
+| Spark 2.2.0   | [download](https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/zoo/analytics-zoo-bigdl_0.8.0-spark_2.2.0/)|
+| Spark 2.3.1   | [download](https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/zoo/analytics-zoo-bigdl_0.8.0-spark_2.3.1/)|
+| Spark 2.4.3   | [download](https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/zoo/analytics-zoo-bigdl_0.8.0-spark_2.4.3/)|
```
Comment (hkvision) [inline docs/docs/release-download.md]: Why aren't they linked to the exact zip file but a web page?

Comment (helenlly) [reply docs/docs/release-download.md]: per dongjie suggested...

Comment (helenlly) [reply docs/docs/release-download.md]: @glorysdj @hkvision  pls sync and let me know your decision

Comment (hkvision) [reply docs/docs/release-download.md]: @glorysdj My opinion is link to the inner folder of 0.6.0-S, i.e. https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/zoo/analytics-zoo-bigdl_0.8.0-spark_2.4.3/0.6.0-SNAPSHOT/

Comment (helenlly) [reply docs/docs/release-download.md]: updated @hkvision

**Comment by yangw1234** [comment]
LGTM

**Pull request merged**